packages/server-ct/src/server-ct.ts

```diff
@@ -58,8 +59,8 @@ export class Server {
     return app
   }
 
-  createRoutes (...args) {
-    return require('./routes-ct').apply(null, args)
+  createRoutes (...args: unknown[]) {
```

Comment:
Can we properly type these args? There's only one usage of `createRoutes`. The `app` and `config` and `project` are passed in inside of `Sever#open`... and have fairly defined business domain concepts

Reply:
In general, `config` is referenced throughout this file, and I'd love to get type completion for it. We use the config object more than anything to grab data from it and I always forget what's on the schema.